lib/api/unittest/CPersistenceManagerTest.cc

```diff
@@ -294,6 +294,80 @@ BOOST_FIXTURE_TEST_CASE(testDetectorBackgroundPersistStaticsConsistency, CTestFi
     this->foregroundBackgroundCompAnomalyDetectionAfterStaticsUpdate("testfiles/new_mlfields_over.conf");
 }
 
+BOOST_FIXTURE_TEST_CASE(testBackgroundPersistCategorizationConsistency, CTestFixture) {
+
+    static const std::string JOB_ID{"job"};
+    static const std::string FIRST_INPUT{
+        "{ \"message\": \"the quick brown fox jumped over the lazy dog\" }\n"};
```

Comment:
Come on Dave! Where's the originality? The inspiration! ;-) 

We could always pick something from http://clagnut.com/blog/2380/#Perfect_pangrams_in_English_.2826_letters.29 :-D